src/simplewallet/password_container.cpp

```diff
@@ -48,24 +48,25 @@ namespace tools
   {
     bool is_cin_tty();
   }
-
+  // deleted via private member
   password_container::password_container()
     : m_empty(true)
   {
-  }
 
-  password_container::password_container(std::string&& password)
-    : m_empty(false)
-    , m_password(std::move(password))
+  }
+  password_container::password_container(const std::string m_wallet_file)
```

Comment:
Also, I don't like that a wallet file name is passed to that. It's got no business with that. Apparently that's just to check if it's empty, so it should be a bool that's passed, not a string to check emptiness.